marcCollection_2019-09-23.sql

```diff
@@ -0,0 +1,58 @@
+# ************************************************************
+# Sequel Pro SQL dump
+# Version 4541
+#
+# http://www.sequelpro.com/
+# https://github.com/sequelpro/sequelpro
+#
+# Host: 127.0.0.1 (MySQL 5.7.27)
+# Database: marcCollection
+# Generation Time: 2019-09-23 13:13:44 +0000
+
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+
+# Dump of table Shoes
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `Shoes`;
+
+CREATE TABLE `Shoes` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `Name` varchar(255) NOT NULL DEFAULT '',
+  `Brand` varchar(255) NOT NULL DEFAULT '',
+  `Primary Colour` varchar(255) NOT NULL DEFAULT '',
+  `Release Year` year(4) NOT NULL,
```

Comment:
best practice is no spaces in table names and camelCase

Reply:
Fixed